lib/tasks/demo_data_helper.rb

```diff
@@ -32,7 +32,12 @@ def humanize_prosecution_case(pcase)
   def case_details_hash(urn)
     pc = prosecution_cases_by_reference(urn).first
 
-    pc.defendants.map do |d|
+    case_details = defendant_offence_details_for(pc)
+    case_details << cracked_ineffective_trial_sentences_for(pc)
+  end
+
+  def defendant_offence_details_for(prosecution_case)
+    prosecution_case.defendants.map do |d|
```

Comment:
could the 'd' in the placeholder be written as a full word?

Reply:
thats done @chris-groves